### PR TITLE
Adds the possibility of generating proportional exome read counts to Ex...

### PR DESCRIPTION
**Pull request by vruano**
...omeReadCounts tool.

Addresses functional issue #434.

### CODE REVIEW

**Comment by vruano** [comment]
@akiezun please review.

**Comment by vruano** [comment]
Sorry, many of the differences in ExomeReadCounts are just due to reformatting (me + automatic by IntelliJ). You can skip confidently differences in java-docs, output file header composers and the GroupBy enum has been moved to the end of the file (it has not been changed at all).

File: src/main/java/org/broadinstitute/hellbender/tools/exome/ExomeReadCounts.java [hunk 1]
```diff
@@ -582,5 +569,114 @@ private ReadGroupCountColumns(final ExomeReadCounts tool) {
...
+
+        /**
+         * Transforms and composes the string representation of the output value.
+         *
+         * @param value       the output matrix value.
+         * @param columnTotal the corresponding column total.
+         * @return never {@code null}.
+         */
+        public String apply(final int value, final long columnTotal) {
+            return operator.apply(value, columnTotal);
```
Comment (akiezun) [inline src/main/java/org/broadinstitute/hellbender/tools/exome/ExomeReadCounts.java]: add value checks (at least this: value >=0, value <= total, total > 0)

**Comment by akiezun** [comment]
one small thing. otherwise :+1:

File: src/main/java/org/broadinstitute/hellbender/tools/exome/ExomeReadCounts.java (hunk 1 above)
Comment (vruano) [inline src/main/java/org/broadinstitute/hellbender/tools/exome/ExomeReadCounts.java]: The total can actually be 0. In this case PCOV would result in NaN values which I believe is a reasonable output in this case. Thus value >= 0, value <= total are enough.

**Pull request merged**